### PR TITLE
BookSuggestion

### PR DESCRIPTION
**Pull request by MaaooVaass**
## Summary 
The Book Suggestions model is created

## Card 
https://trello.com/c/vQaWc47t/26-crear-modelo-de-booksuggestion

### CODE REVIEW

File: spec/factories/book_suggestions.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :book_suggestion do
+    synopsis { "MyString" }
+    price { 1.5 }
+    author { "MyString" }
+    title { "MyString" }
+    link { "MyString" }
+    publisher { "MyString" }
+    year { "MyString" }
+    user { nil }
```
Comment (fedeest) [inline spec/factories/book_suggestions.rb]: User the Faker Gem for the factories of models. That way you can get random values.

**Review by fedeest — CHANGES_REQUESTED** [review]
Also, add the .DS_Store files to the .gitignore

File: app/models/book_suggestion.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class BookSuggestion < ApplicationRecord
+  belongs_to :user
```
Comment (lmasello) [inline app/models/book_suggestion.rb]: Don't forget to add the inverse relation. This is, the User model should have a has_many book_suggestions

**Review by lmasello — CHANGES_REQUESTED** [review]

**Comment by lmasello** [comment]
The PR description should be in English too.

File: app/models/user.rb [hunk 3]
```diff
@@ -2,4 +2,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable, :trackable
   include DeviseTokenAuth::Concerns::User
   has_many :rents, dependent: :destroy
+  has_many :book_suggestion, dependent: :destroy
```
Comment (lmasello) [inline app/models/user.rb]: book_suggestions

**Review by lmasello — CHANGES_REQUESTED** [review]
Be careful, you are adding .DS_Store files

**Comment by lmasello** [comment]
@MaaooVaass Be careful, you are adding .DS_Store files

**Review by fedeest — APPROVED** [review]

**Review by lmasello — APPROVED** [review]

**Pull request merged**